CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.7.0-dev.9 (2016-03-18)
+- Support labels on networks
```

Comment:
It's ok. We do only support network creation, no modification. It's implicit then.